tools/contractdeployer/main/main.go

```diff
@@ -10,7 +11,20 @@ import (
 
 func main() {
 	log.SetLogLevel(log.DisabledLevel)
-	config := contractdeployer.ParseConfig()
+	// config := contractdeployer.ParseConfig()
```

Comment:
it's back :)

Reply:
Ah damn! I even made a mental note to clean it up this time haha. It is v handy, but seems a bit hacky to leave it here (I don't think these values will work forever).

I've got a branch that has some dev scripts for doing ad-hoc stuff like this, should try and polish that up and PR it, would be useful for testing when you want to check something on testnet or use debugger on RPC responses.